### ISSUE TITLE
unzip error

### ISSUE THREAD

**Issue by SlyDave** [antonioribeiro/countries#54]
```
david$ composer require pragmarx/countries
Using version ^0.1.1 for pragmarx/countries
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 4 installs, 0 updates, 0 removals
  - Installing commercie/currency (1.1.0): Downloading (100%)         
  - Installing symfony/yaml (v3.4.2): Downloading (100%)         
  - Installing mledoze/countries (1.8.0): Downloading (100%)         
  - Installing pragmarx/countries (v0.1.1): Downloading (100%)         
    Failed to execute unzip -qq  '/Users/david/Projects/sentry-defence/vendor/pragmarx/countries/36d386babf999bfe85247e40ed0e03c1' -d '/Users/david/Projects/sentry-defence/vendor/composer/5cdc8b03'

replace /Users/david/Projects/sentry-defence/vendor/composer/5cdc8b03/antonioribeiro-countries-7941fd2/tests/phpunit/ValidationTest.php? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)

    The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
    Unzip with unzip command failed, falling back to ZipArchive class
commercie/currency suggests installing drupal/currency (Integrates this package with Drupal.)
Writing lock file
Generating optimized autoload files
```

Running on macOS Sierra 10.12.6 (16G1036)

**Comment by pra-ei** [comment]
Have the same issue.
Happened both on Homestead (Ubuntu 16.04.3 LTS), and MacOS Sierra.

**Comment by SylvesterNielsen** [comment]
Same issue here on Native Ubuntu 16.04.03 (PHP7.1.10), Homestead (Ubuntu 16.04.3 LTS, PHP7.1.10), and Windows 10 (PHP7.1.11)

**Comment by SylvesterNielsen** [comment]
The issue seems to be related to this line:
https://github.com/antonioribeiro/coollection/blob/master/upgrade-collect.sh#L141

> -- Renaming namespace from Illuminate to Tightenco...  
> src/vendor/tightenco/collect/src/Tightenco/Collect - src/vendor/tightenco/collect/src/Illuminate  
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory
> sed: can't read : No such file or directory

**Comment by SylvesterNielsen** [comment]
The issue is related to the phpunit folders in tests. On systems which is file not casesensitive composer are unable to unzip the content due to the conflict of the two ValidationTest.php files.

For some reason, it seems to work running composer install on native windows and on native Ubuntu.

**Comment by alex-arriaga** [comment]
Same issue here. By any chance, someone knows how to fix it?

**Comment by pra-ei** [comment]
You need to have caseSensitive filesystem to use this package, otherwise it will fail due to having similar names on paths related to test.

**Comment by sandofvega** [comment]
Same :|

**Comment by antonioribeiro** [comment]
This is it running on my macOS Sierra:

![image](https://user-images.githubusercontent.com/3182864/35769974-875d80e4-08fa-11e8-91b1-bfaab04c33b1.png)

@SlyDave, you are, somehow, downloading 0.1.1, but the last version is 0.5.*, I know this thread is kind of old, but is it still happening?

**Comment by aphoe** [comment]
@antonioribeiro yes, it is still happening. Iam working with a Windows system. The work around was to install the `dev-master`  version

**Comment by tekkenking** [comment]
@antonioribeiro yes,this is also the problem i am facing. Please help here.. I am on windows 10 system. Thanks

**Comment by antonioribeiro** [comment]
Tests folders doesn't exists anymore, there's only one test file. 
I cannot test this on Windows, I don't have a Windows OS anymore.
Could someone, please, try it again and give us some more information?
